### ISSUE TITLE
Follow Web Accessibility standards

### ISSUE THREAD

**Issue by chmey** [chmey/py-week-kanban#9]
kanweek should be usable by everyone. Therefore it is important to follow standards for accessibility.

Currently, it is not supported in all parts of the application.

https://reactjs.org/docs/accessibility.html gives an overview about providing accessibility.

Where needed, such changes must be applied.

Make sure the app follows at least one of the checklists provided on the website above.
